cmd/snap/cmd_quota_test.go

```diff
@@ -172,6 +202,56 @@ func makeChangesHandler(c *check.C) func(w http.ResponseWriter, r *http.Request)
 	}
 }
 
+func (s *quotaSuite) TestParseQuotas(c *check.C) {
+	for _, testData := range []struct {
+		maxMemory string
+		cpuMax    string
+		cpuSet    string
+		threadMax string
+
+		// Use the JSON representation of the quota, as it's easier to handle in the test data
+		quotas string
+		err    string
+	}{
+		{maxMemory: "12KB", quotas: `{"memory":12000,"cpu":{},"cpu-set":{}}`},
+		{cpuMax: "12x40%", quotas: `{"cpu":{"count":12,"percentage":40},"cpu-set":{}}`},
```

Comment:
Could you please add a case for cpuMx with just "40%" as an input?

Reply:
The spec currently says "--thead", I asked in the gdoc for Samueles input on this.